.circleci/config.yml

```diff
@@ -264,9 +270,21 @@ jobs:
             - frontend/src/autogen/proto.js
             - lib/streamlit/proto
 
+  # The following jobs inherit everything from python-3.7.4. In a few cases,
+  # there are steps that are skipped based on the name of the current job (see,
+  # e.g., "Run frontend tests").
+  python-3.8:
+    <<: *job-template
+    docker:
+      - image: circleci/python:3.8.1
+  python-3.5:
+    <<: *job-template
+    docker:
+      - image: circleci/python:3.5.9
+
   cypress:
     docker:
-      - image: circleci/python:3.7.4-stretch
+      - image: circleci/python:3.8.1
```

Comment:
Why run cypress with latest python instead of 3.7.4?

Reply:
I was just trying to move to using the latest version since I expected that we'd be dropping testing of 3.7.4 in favor of testing the oldest and newest versions we supported, and it seemed odd to keep using an in-between version here. However, I've reverted this since this change resulted in font changes as well, which were undesirable for visual diffing.